### ISSUE TITLE
time-delayed hint

### ISSUE THREAD

**Issue by donpdonp** [KenEucker/biketag-website#6]
the current bike tag photo has got me stumped. it might be useful to have a mechanism that kicks in after a set amount of time, such as a written hint, to keep the game moving forward.

**Comment by donpdonp** [comment]
poking around the site a bit more i see biketag.org does not show hints yet biketag.org/?count=10 shows hints. seems like the UI should work a little differently, such as a 'click here to show hint' on either page url.

**Comment by KenEucker** [comment]
Hey @donpdonp! I just saw these issues you created. Thanks for the feedback! 

You're touching on something here that I've wanted to put more work into as a next step. I think that improving the UI on the site is good, I had planned to do a time delayed comment on Reddit for the hint but that translates to the site pretty well so we can do it there too.

I know we've moved on from that last tag you mentioned, I'm stumped with the current one now!
